tests/integration-tests/tests/intel_hpc/test_intel_hpc.py

```diff
@@ -0,0 +1,83 @@
+from tests.common.assertions import assert_no_errors_in_logs
+from tests.common.schedulers_common import get_scheduler_commands
+from tests.common.utils import fetch_instance_slots
+
+
+@pytest.mark.regions(["us-east-1"])
+@pytest.mark.instances(["c5n.18xlarge"])
+@pytest.mark.oss(["centos7"])
+@pytest.mark.schedulers(["sge"])
+def test_intel_hpc(region, scheduler, instance, os, pcluster_config_reader, clusters_factory, test_datadir):
+    """
+    Test Intel Cluster Checker
+    """
+    slots_per_instance = fetch_instance_slots(region, instance)
+    cluster_config = pcluster_config_reader()
+    cluster = clusters_factory(cluster_config)
+    remote_command_executor = RemoteCommandExecutor(cluster)
+    scheduler_commands = get_scheduler_commands(scheduler, remote_command_executor)
+    _test_intel_clck(remote_command_executor, scheduler_commands, slots_per_instance, test_datadir)
+
+    assert_no_errors_in_logs(remote_command_executor, ["/var/log/sqswatcher", "/var/log/jobwatcher"])
+
+
+def _test_intel_clck(remote_command_executor, scheduler_commands, slots_per_instance, test_datadir):
+    # Install Intel Cluster Checker CLCK Master
+    logging.info("Installing Intel Cluster Checker")
+    remote_command_executor.run_remote_script(str(test_datadir / "install_clck.sh"), hide=False)
+
+    # Install Intel Cluster Checker CLCK Compute
+    result = scheduler_commands.submit_script(
+        str(test_datadir / "install_clck_compute.sh"), slots=2 * slots_per_instance
+    )
+    job_id = scheduler_commands.assert_job_submitted(result.stdout)
+    scheduler_commands.wait_job_completed(job_id)
+    scheduler_commands.assert_job_succeeded(job_id)
+
+    # Create nodefile
+    # ip-172-31-15-31  # role: head
+    # ip-172-31-12-237  # role: compute
+    # ip-172-31-8-49  # role: compute
+    remote_command_executor.run_remote_command("echo $HOSTNAME | awk '{print $1 \" # role: head\" }' > nodefile")
+    remote_command_executor.run_remote_command(
+        "qhost | tail -n +4 | awk '{print $1 \" # role: compute\" }' >> nodefile"
+    )
+    result = remote_command_executor.run_remote_command("cat nodefile | wc -l")
+    assert_that(result.stdout).contains("3")
+
+    # Setup network interface
+    # <!-- This tag can be used to set the network interface used for
+    #      accumulating data collected on-demand.
+    # -->
+    # <!--
+    # <network_interface>ens5</network_interface>
+    # -->
+    # /opt/intel/clck/2019.3.5/etc/clck.xml
+    remote_command_executor.run_remote_command(
+        "sudo cp ~/clck.xml /opt/intel/clck/2019.3.5/etc/clck.xml", additional_files=[str(test_datadir / "clck.xml")]
+    )
+
+    # Run Cluster Checker
+    result = remote_command_executor.run_remote_script(str(test_datadir / "run_clck.sh"))
+    try:
+        assert_that(result.stdout).contains("Overall Result: PASS")
+    except AssertionError as e:
+        logging.error(remote_command_executor.run_remote_command("cat clck_results.log"))
```

Comment:
Probably a .stdout is missing after run_remote_command

Reply:
hmm... it works this way, but I've added it anyways